src/main/java/seedu/duke/command/timetable/TimeTableCommand.java

```diff
@@ -0,0 +1,108 @@
+package seedu.duke.command.timetable;
+
+import seedu.duke.command.Command;
+import seedu.duke.command.CommandResult;
+import seedu.duke.data.Module;
+import seedu.duke.data.ModuleManager;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+
+import static seedu.duke.ui.TextUi.DIVIDER_LINE;
+
+public class TimeTableCommand extends Command {
```

Comment:
Command classes should only call methods or store and pass information, and should not have standalone methods. Ideally, your timetable command would call 'TimeTable.generateWeekTable()' (and others) and not have its own method for doing so.

Reply:
Ok, will make amends